building/build-debs/homeworld-admin-tools/src/access.py

```diff
@@ -191,8 +185,19 @@ def dispatch_kubectl(*params: str):
     call_kubectl(params, False)
 
 
+def ssh_foreach(node_kind: str, *params: str):
+    config = configuration.get_config()
+    valid_node_kinds = configuration.Node.VALID_NODE_KINDS
+    if not (node_kind == "node" or node_kind in valid_node_kinds):
+        command.fail("usage: spire foreach {node," + ",".join(valid_node_kinds) + "} command")
+    for node in config.nodes:
+        if node_kind == "node" or node.kind == node_kind:
+            ssh.check_ssh(node, *params)
```

Comment:
Maybe it would be good to have a header here saying which node is getting a command run on it?

(Maybe it would work best to use some of the Operation machinery?)